docs/quick-start/new-project/README.md

```diff
@@ -7,17 +7,17 @@ Next, install Mira, the AWS CDK and other dependencies. Edit your `package.json`
 
 ```json
  "dependencies": {
-    "@aws-cdk/aws-cloudformation": "1.67.0",
-    "@aws-cdk/aws-lambda-event-sources": "1.67.0",
-    "@aws-cdk/aws-rds": "1.67.0",
-    "@aws-cdk/aws-s3-deployment": "1.67.0",
-    "@aws-cdk/aws-secretsmanager": "1.67.0",
-    "@aws-cdk/core": "1.67.0",
-    "aws-cdk": "1.67.0",
+    "@aws-cdk/aws-cloudformation": "1.89.0",
+    "@aws-cdk/aws-lambda-event-sources": "1.89.0",
+    "@aws-cdk/aws-rds": "1.89.0",
+    "@aws-cdk/aws-s3-deployment": "1.89.0",
+    "@aws-cdk/aws-secretsmanager": "1.89.0",
+    "@aws-cdk/core": "1.89.0",
+    "aws-cdk": "1.89.0",
     "mira": "^1.3.0"
```

Comment:
This will be version 1.4.2 of Mira. See https://www.npmjs.com/package/mira
Maybe it's a bit premature, and that change should be part of the release? But if so, we need to make sure the release instructions include this change.

Reply:
Bumped the version number to 1.4.2. Makes sense to do that now so it's not forgotten later.